test/hotspot/jtreg/runtime/whitebox/TestWBDeflateIdleMonitors.java

```diff
@@ -61,12 +61,11 @@ public static class InflateMonitorsTest {
         static WhiteBox wb = WhiteBox.getWhiteBox();
         public static Object obj;
 
-        public static void main(String args[]) {
+        public static void main(String args[]) throws Exception {
             obj = new Object();
             synchronized (obj) {
-                // HotSpot implementation detail: asking for the hash code
-                // when the object is locked causes monitor inflation.
-                if (obj.hashCode() == 0xBAD) System.out.println("!");
```

Comment:
Why was this deleted from the test? `get_next_hash()` can still
return this value so it's useful to know if a bad hash code made it
out this far...